app/models/conversion_host.rb

```diff
@@ -91,7 +97,7 @@ def verify_credentials(auth_type = nil, options = {})
   #  - The number of concurrent tasks has not reached the limit.
   #
   def eligible?
-    source_transport_method.present? && verify_credentials && check_concurrent_tasks
+    source_transport_method.present? && authentication_check('v2v').first && check_concurrent_tasks
```

Comment:
@djberg96 could you use `authentication_status_ok?` instead?  Then you wouldn't need the logic to not check the credentials if they were checked recently.  It would just rely on the scheduled auth check.

Reply:
@agrare Good call, I've updated the PR. I think this addresses an issue that @kbrock had as well.